ci/jupyter_notebook_config.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from bookstore import BookstoreContentsArchiver, BookstoreSettings
+
+# jupyter config
+# At ~/.jupyter/jupyter_notebook_config.py for user installs
+# At __ for system installs
+c = get_config()
+
+c.NotebookApp.contents_manager_class = BookstoreContentsArchiver
+
+c.BookstoreSettings.workspace_prefix = "ci-workspace"
+
+# If using minio for development
+c.BookstoreSettings.s3_endpoint_url = "http://127.0.0.1:9000"
```

Comment:
Do we also need to test https:// ? Probably doesn't matter at this stage, but to keep in mind for later tests.

Reply:
We can visit that later. I'm going a little overboard with integration tests here already. 😅

Reply:
Works for me ;-)